cicd/crabserver_pypi/run.sh

```diff
@@ -14,9 +14,9 @@ mkfifo /data/srv/state/crabserver/crabserver-fifo
 # to pipe. It is safe because only single process can read from pipe at the time
 cat /data/srv/state/crabserver/crabserver-fifo &
 
-#start the service
-#export CRYPTOGRAPHY_ALLOW_OPENSSL_102=true
-/data/start.sh -c
+# start the service
+# manage.py required service name, anything is fine because we never use it.
```

Comment:
why do you keep this argument if we never use it ?
(N.B. "required" or "requires" ? past or present ?)

Reply:
Ah sorry. I misunderstand the question.
The SERVICE arg is only use in publisher. It is already setup by runContainer.sh script in TW but not the rest.  
At the time I dev manage.py, make it as require arguments is much simpler to code.

But let me try again to make it optional. It might be easier than I thought.

Reply:
Well, turn out it is easier than I thought :)